### ISSUE TITLE
Multiple accounts dont show up in pick list

### ISSUE THREAD

**Issue by t8or** [tymscar/runelite-1password-plugin#4]
Mac OS 13.4.1 (22F82)
Latest 1Password Plugin version
1Password for Mac 8.10.8 (81008045)
Runelite 2.6.3

When one 1Password file with https://runescape.com exists, the plugin works like expected.

When multiple entries in 1Password exist with https://runescape.com exist, the picker drop down field is blank:
<img width="621" alt="image" src="https://github.com/tymscar/runelite-1password-plugin/assets/17255992/09517bdf-f288-4bd2-bf6f-01ff150298e5">

Would love to be able to use this plugin, but right now am big sad :(

**Comment by t8or** [comment]
Bump :( It is also happening on Windows

**Comment by tymscar** [comment]
Fixed. Sorry it took so long!

**Comment by t8or** [comment]
Oscar, you are a legend, thank you!! I'll test later today.

**Comment by tymscar** [comment]
Unless you use it from this repo here, you need to wait a bit. It has not been merged into the official repo yet. Waiting on this https://github.com/runelite/plugin-hub/pull/8758

**Comment by t8or** [comment]
Ah yes. Good ol' plug-in hub. We shall wait. Thank you!!

**Comment by tymscar** [comment]
Finally got merged! You can give it a go now!
